daisy_workflows/image_import/inspection/src/boot_inspect/inspectors/os/linux.py

```diff
@@ -21,17 +21,8 @@
 import boot_inspect.system.filesystems
 
 
-class LegacyFingerprint:
-  """Identifies systems using pre-systemd metadata files.
-
-  Prior to systemd's specification of /etc/os-release,
-  https://www.freedesktop.org/software/systemd/man/os-release.html,
-  there was no universal standard for specifying system metadata. This
-  class encodes which files were used during that period.
-
-  A positive match is based on:
-    1. The existence of a standard metadata file
-    2. The non-existance of derivative metadata files.
+class SentinelFileMatcher:
```

Comment:
First time I hear this term (sentinel file). Either provide more background on the meaning of this term, or consider renaming to something more easily understandable.

Reply:
It's a reference to [1]. Any other naming ideas?

1. https://en.wikipedia.org/wiki/Sentinel_value

Reply:
done!